### DISCUSSION TITLE
about _commonjsHelpers.b8add541.js

### DISCUSSION THREAD

**Discussion by DrAugus** [vuepress/core#974] [Q&A]
## error

Can't get this file `_commonjsHelpers.b8add541.js`

![image](https://user-images.githubusercontent.com/34206897/176808905-d5935bf0-e648-4489-98c3-d7d1489be6c6.png)

## [repo](https://github.com/DrAugus/draugus.github.io) 

![image](https://user-images.githubusercontent.com/34206897/176808928-f9be047e-30de-4828-8fdb-947c5b88a05a.png)

## url 

- [error page1](https://draugus.github.io/life/food)
- [error page2](https://draugus.github.io/life/game/genshin/wish)

**Comment by Mister-Hope** [comment]
I think this is kind of GitHub issue. Contact GitHub support why you cannot fetch this file.

**Reply by DrAugus** [reply]
gh的Jekyll问题 把_开头的文件剔除了
